setup.py

```diff
@@ -27,6 +27,7 @@
         ],
     },
     tests_require=[
+        'setuptools>=17.1',
```

Comment:
Sorry, that was a test. reverting

On Wed, Oct 7, 2015 at 6:00 PM, Laurent Raufaste <notifications@github.com>
wrote:

> In setup.py
> <https://github.com/percolate/dripconfig/pull/4#discussion_r41453535>:
>
> > @@ -27,6 +27,7 @@
> >          ],
> >      },
> >      tests_require=[
> > +        'setuptools>=17.1',
>
> nooooooooooooo
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/percolate/dripconfig/pull/4/files#r41453535>.
>



-- 
Xavier Lapointe
Python Developer
Get inspired. See what's new on the Percolate blog: blog.percolate.com